### ISSUE TITLE
wiki

### ISSUE THREAD

**Issue by polymathuk** [smehrbrodt/nextcloud-libreoffice-online#5]
Apologies if this is not the correct place to write this. I tried to set this project up as per the wiki. which I followed to the letter using two VMs, one for NC and one for LO. The NC side went without a problem, but the LO I cannot connect to. The wiki says that:
"The file loolwsd.xml in that directory is the main config file....
Edit it to your liking...."
Is it necessary to edit anything in this file straight out of the box? I cannot connect to the LibreOffice Online admin dashboard using an adapted form of the URL in the wiki (from another machine on the same network). http://192.168.1.19:9980/loleaflet/dist/admin/admin.html
I installed lynx on my VM and tried:
http://localhost:9980/loleaflet/dist/admin/admin.html
but no dice.
Is there something really obvious that I'm doing wrong? The message I'm getting in the browser is "192.168.1.19 refused to connect." I tried setting SSL to false in the xml but this didn't help.
Last few lines of the docker log:
online_1  | wsd-00015-00016 2019-06-15 11:52:22.586204 [ prisoner_poll ] TRC  Poll completed with 0 live polls max (5000ms)(timedout)| ./net/Socket.hpp:521
online_1  | wsd-00015-00016 2019-06-15 11:52:22.586279 [ prisoner_poll ] TRC  Poll start| ./net/Socket.hpp:511

**Comment by polymathuk** [comment]
This is most peculiar. Further to my last post, I installed telnet on the same VM that hosts LO. When I try:
  telnet localhost 9980
it says connection closed (immediately).
I looked at the iptables for a hint, but nothing obvious though I'm not an expert on this and there does seem to be a fair bit of dropping packets. It looks to my untrained eye that the docker has its own ip address (172.18.0.2) which I can ping from the VM but telnet fails to connect. As does lynx (both on the same VM)

I'm opening this issue because an out-of-the-box install fails to work. This could be me, or it could be a general problem... I would be very grateful for any help.

**Comment by smehrbrodt** [comment]
Can you provide your full logs (in a pastebin, or as attachment)?

**Comment by polymathuk** [comment]
Well, this is embarrassing(!) - although you'll be very happy to learn that the issue was of my own causing.
I was running both NC & LO in separate VMs which I clone from a debian9 (installed) image. Your suggestion about the logs made me think, so I found them and inspected them. One line that jumped out at me was 'out of memory' - and I realised the VMs only have 256MB each... I fixed that problem by upping the RAM to 2GB in LO. NC runs fine with 256MB. I am monitoring both in the VM manager so that the docker RAM figures are properly accounted for. LO tops out at 1674MB with a document open. However, this still didn't fix the problem so I had another look at the log - 'no space on volume' - the basic image only has 4GB. 8GB is the minimum I found that works for LO. Once I had those problems solved, it worked fine.
So I apologise for wasting your time.
However, I would make the following humble suggestion as tips for other experimenters: add resource requirements to the wiki: 
NC disk 2GB / RAM 256MB min
LO disk 8GB / RAM 2GB min
Thanks.
